docs/docfx.json

```
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/docfx/main/schemas/docfx.schema.json",
+  "metadata": [
+    {
+      "src": [
+        {
+          "src": "../src",
+          "files": [
+            "**/*.csproj"
+          ]
+        }
+      ],
+      "dest": "api",
+      "properties": {
+        "ProduceReferenceAssembly": "true"
+      }
+    }
+  ],
+  "build": {
+    "content": [
+      {
+        "files": [
+          "**/*.{md,yml}"
+        ],
+        "exclude": [
+          "_site/**"
+        ]
+      }
+    ],
+    "resource": [
+      {
+        "files": [
+          "images/*"
+        ]
+      }
+    ],
+    "output": "_site",
+    "template": [
+      "default",
+      "modern"
+    ],
+    "globalMetadata": {
+      "_appFaviconPath": "images/favicon.svg",
+      "_appLogoPath": "images/favicon.svg",
+      "_appName": "Apache Arrow .NET",
+      "_appTitle": "Apache Arrow .NET",
+      "_appFooter": "© The Apache Software Foundation",
```

Could you also add "year"?

```suggestion
      "_appFooter": "© 2018 The Apache Software Foundation",
```